ext/node/polyfills/http.ts

```diff
@@ -1419,6 +1419,20 @@ export class ServerResponse extends NodeWritable {
     this.#headers[name] = value;
     return this;
   }
+  appendHeader(name: string, value: string | string[]) {
+    if (Array.isArray(value)) {
+      this.#hasNonStringHeaders = true;
+    }
+    if (this.#headers[name] === undefined) {
+      this.#headers[name] = value;
+    } else {
+      if (!Array.isArray(this.#headers[name])) {
+        this.#headers[name] = [this.#headers[name]];
+      }
+      this.#headers[name].push(value);
+    }
```

Comment:
```suggestion
    if (this.#headers[name] === undefined) {
      this.#headers[name] = value;
    } else {
      this.#headers[name] = [].concat(this.#headers[name], value);
    }
```

nit: trick if you to reduce if statements. `concat` returns a new array but in this case perf is not an issue considering header values are limited.

Reply:
If this API is used in a project using `.concat()` is an additional allocation per request though.